### ISSUE TITLE
Jellyfin Media Player Crash on Startup (Ubuntu 24.10)

### ISSUE THREAD

**Issue by ajsb85** [jellyfin/jellyfin-media-player#819]
Jellyfin Media Player crashes immediately on startup with a "QSocketNotifier: Can only be used with threads started with QThread" error and a Wayland assertion failure.

**To Reproduce**

1. Launch Jellyfin Media Player from the command line using `jellyfinmediaplayer`.
2. Observe the error messages printed to the console and the application crashing.

**Expected behavior**

Jellyfin Media Player should launch successfully and display the user interface.

**Screenshots**

N/A (Crash happens before any UI is displayed)

**Desktop (please complete the following information):**

*   OS: Ubuntu 24.10
*   Kernel Version: 6.11.0-9-generic
*   Installation Method: 
    * Installed .deb package (`sudo apt install ./jellyfin-media-player_1.11.1-1_amd64-noble.deb`)
    * Installed Flatpak package (`com.github.iwalton3.jellyfin-media-player.flatpakref`)
*   Qt Version: 5.15.13
*   Graphics Card:  NVIDIA Corporation GM206 [GeForce GTX 950] (rev a1)
*   Graphics Driver: 4.6.0 NVIDIA 560.35.03

**Additional context**

*   The crash is accompanied by the following error messages:
    *   `QSocketNotifier: Can only be used with threads started with QThread`
    *   `jellyfinmediaplayer: ../src/wayland-eglsurface.c:2562: wlEglDestroySurface: Assertion 'wl_list_empty(&surface->ctx.streamImages)' failed.`
*   The application log file (`/home/ajsb85/.local/share/jellyfinmediaplayer/logs/jellyfinmediaplayer.log`) contains several warnings, including:
    *   Missing `idmatcher` element in input mapping files.
    *   `QLocalSocket::ServerNotFoundError` related to LIRC.
    *   Warnings about `QtWebEngine::initialize()` being called after `QCoreApplication` is created and the `Qt::AA_ShareOpenGLContexts` attribute not being set before `QCoreApplication` creation.
    *   A critical error about a missing slot for the `recreateRpiUI` host command (likely unrelated as I am not using a Raspberry Pi).
*   I have not modified any custom CSS.
*   I have attached the complete log file to this issue. [jellyfinmediaplayer.log](https://github.com/user-attachments/files/18021477/jellyfinmediaplayer.log)

**Possible causes and attempted solutions:**

*   I have updated Jellyfin Media Player to the latest version (1.11.1).
*   I have tried reinstalling Qt, but the issue persists.
*   I have searched the Jellyfin Media Player bug tracker and forums but haven't found any reports of this specific issue.
*   I have tried installing both the .deb package and the Flatpak version, but the crash occurs with both.

I suspect the crash might be related to threading issues within the application, incompatibility with Wayland, or a conflict with my graphics drivers. I would appreciate any assistance in diagnosing and resolving this problem.

**Comment by CastleBravoMegatons** [comment]
Yep its something with the graphics driver. The app does not crash when using Intel Integrated Graphics. Gif Related
![Nvidia Fuck You](https://c.tenor.com/BNA0KXjX-1AAAAAd/tenor.gif)

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
